### PR TITLE
Use constant

### PR DESCRIPTION
**Pull request by HuKeping**
Signed-off-by: Hu Keping <hukeping@huawei.com>

### CODE REVIEW

**Comment by riyazdf** [comment]
Jenkins, test this please.

**Comment by riyazdf** [comment]
Thanks for the cleanup! :+1: 

`client/client.go` and `tuf/client/client.go` have some hardcoded `"targets"` and `"root"` values, and `tuf.go` also seems to have a hardcoded `"root"` -- would you mind also changing these to the `data` constants so that these files are completely using constants for roles?

**Comment by HuKeping** [comment]
I'll update it later, thanks @riyazdf

**Comment by HuKeping** [comment]
updated @riyazdf

File: client/client.go [hunk 1]
```diff
@@ -341,7 +341,7 @@ func addChange(cl *changelist.FileChangelist, c changelist.Change, roles ...stri
 
 // AddTarget creates new changelist entries to add a target to the given roles
 // in the repository when the changelist gets applied at publish time.
-// If roles are unspecified, the default role is "targets".
+// If roles are unspecified, the default role is data.CanonicalTargetsRole.
```
Comment (riyazdf) [inline client/client.go]: nit: this could stay as "targets" since it's just a comment, but this is fine too

**Comment by riyazdf** [comment]
thanks for updating this!  I have a small nit comment but LGTM

File: client/client.go (hunk 1 above)
Comment (HuKeping) [inline client/client.go]: Have changed it back :)

**Comment by endophage** [comment]
LGTM! Thanks!

**Pull request merged**